FirebaseInstallations/Source/Library/Public/FIRInstallations.h

```diff
@@ -14,39 +14,101 @@
  * limitations under the License.
  */
 
-// TODO: Add short docs to the API
 #import <Foundation/Foundation.h>
 
-@class FIRInstallationsAuthTokenResult;
+#import <FirebaseInstallations/FIRInstallationsAuthTokenResult.h>
+
 @class FIRApp;
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern NSString *const FIRInstallationIDDidChangeNotification;
+/** A notification with this name is sent each time an installation is created or deleted. */
+FOUNDATION_EXPORT const NSNotificationName FIRInstallationIDDidChangeNotification;
 
+/**
+ * An installation ID handler block.
+ * @param identifier The instalation ID string if exists or @c nil otherwise.
+ * @param error The error when @c identifier==nil or @c nil otherwise.
+ */
 typedef void (^FIRInstallationsIDHandler)(NSString *__nullable identifier,
                                           NSError *__nullable error)
     NS_SWIFT_NAME(InstallationsIDHandler);
 
+/**
+ * An authorization token handler block.
+ * @param tokenResult An instance of @c FIRInstallationsAuthTokenResult in case of success or @c nil
```

Comment:
Nit: lets use the Swift names, so remove the `FIR` prefix here and below.

Reply:
I assume we will need to change `[FIRInstallations authTokenForcingRefresh:completion:]` as well then?